docs/pages/eas-update/eas-update-with-local-build.mdx

```diff
@@ -0,0 +1,260 @@
+      <StatusBar style="auto" />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+});
+```
+
+Then build and run the app again:
+
+<Terminal
+  cmd={[
+    '# iOS: Build and run the app on a local iOS Simulator.',
+    '$ npx expo run:ios --configuration Release',
+    '',
+    '# Android: Build and run the app on a local Android Emulator',
+    '$ npx expo run:android --variant release',
+    '',
+  ]}
+  cmdCopy="npx expo run:ios --configuration Release && npx expo run:android --variant release"
+/>
+
+The app should appear as in the screenshot below.
+
+<ImageSpotlight
+  alt="App running for the first time"
+  src="/static/images/eas-update/getting-started-before-update.png"
+  style={{maxHeight: 600, width: 'auto'}}/>
+</Step>
+
+<Step label="7">
+
+## Publish an update
+
+Now we're ready to publish an update that will include our changes.
+
+After optionally making additional changes to **App.js**, publish an update with the following command:
+
+<Terminal cmd={['$ eas update', '']} cmdCopy="eas update" />
+
+Running the command will prompt the following:
+
+- For the branch to publish on. Select the default branch: `main`.
+- For entering an update message. Select the default `Initial commit` message or type in a custom message.
+
+Once the update is built and uploaded to EAS and the command completes, force close and reopen your app up to two times to download and view the update.
+
+The app should now appear as shown in the screenshot below:
+
+<ImageSpotlight
+  alt="App running update after being restarted twice"
+  src="/static/images/eas-update/getting-started-after-update.png"
+  style={{ maxHeight: 600, width: 'auto' }}
+/>
+
+</Step>
+
+## Testing updates with a debug build of your app
+
+It is possible to [create a debug build that will load updates from EAS instead of loading from a React Native packager](/eas-update/debug-updates/#debugging-of-native-code-while-loading-the-app-through-expo-updates).
+
+To do this for the above app, modify step 4 above as follows:
+
+<Terminal
+  cmd={[
+    '# Set the environment variable needed for native debug builds',
+    '$ export EX_UPDATES_NATIVE_DEBUG=1',
+    '',
+    '# Run prebuild to install and configure the ios and android directories, and install Cocoapods for iOS.',
+    '$ npx expo prebuild',
+    '',
+    '# iOS: Modify the application project so that the application JavaScript',
+    '',
+    '# is bundled into the app for both debug and release builds.',
+    "$ sed -i '' 's/SKIP_BUNDLING/FORCE_BUNDLING/g;' ios/<project name>.xcodeproj/project.pbxproj",
+    '',
+    '# iOS: Build and run the app on a local iOS Simulator.',
+    '$ npx expo run:ios --configuration Debug',
+    '',
+    '# Android: Build and run the app on a local Android Emulator',
+    '$ npx expo run:android --variant debug',
+    '',
+  ]}
+  cmdCopy={null}
+/>
```

Comment:
It would be nice to split those instruction per platform, maybe even we can use Tabs in there.

Reply:
I can work on that separately